src/test/java/de/tudarmstadt/lt/cw/global/CWGlobalTest.java

```diff
@@ -13,6 +13,8 @@
 
 import org.junit.Test;
 
+import de.tudarmstadt.lt.cw.CW.Option;
```

Comment:
Can you please add here tests to other modes as well to make sure that other options also work with the global clustering mode?